x-pack/plugins/endpoint/public/applications/endpoint/store/action.ts

```diff
@@ -4,9 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { ManagementAction } from './managing';
+import { HostAction } from './managing';
 import { AlertAction } from './alerts';
 import { RoutingAction } from './routing';
 import { PolicyListAction } from './policy_list';
 
-export type AppAction = ManagementAction | AlertAction | RoutingAction | PolicyListAction;
+export type AppAction = HostAction | AlertAction | RoutingAction | PolicyListAction;
```

Comment:
noticed that you're renaming your code to 'host' and also 'hosts' on some cases. thoughts?